tsconfig-ts.dependency.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "allowUnreachableCode": false,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "CommonJS",
+    "noEmit": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "ES2019",
+    "resolveJsonModule": true,
+    "sourceMap": true,
+    "baseUrl": "./packages",
+    "paths": {
+      "@tupaia/*": ["*/src"]
```

Comment:
This single config has changed my life ❤️ it allows our ts packages to do source mapping on our internal packages without requiring a rebuild. Super handy!

Reply:
Awesome find man!!!

Reply:
It's also super neat that we just had to update this in only place